client/verta/verta/_cli/registry/list.py

```diff
@@ -55,9 +55,31 @@ def lst_model(filter, output, workspace):
 def lst_model_version(model_name, filter, output, workspace):
     """List all models available.
     """
+    client = Client()
+
     if model_name is None:
         click.echo("Listing versions for all models")
+        registered_model = None
     else:
         click.echo("Listing versions for model {}".format(model_name))
+        registered_model = client.get_registered_model(model_name, workspace)
+
+    model_versions = client.registered_model_versions.with_model(registered_model).find(filter)
+
     click.echo("filter: {}".format(filter))
     click.echo("output: {}".format(output))
+
+    click.echo("result count: {}".format(len(model_versions)))
+    click.echo("Listing all model versions")
```

Comment:
Similar to https://github.com/VertaAI/modeldb/pull/1045#discussion_r458487281
```suggestion
    click.echo()
```